ts/components/conversation/ConversationHeader.tsx

```diff
@@ -86,7 +86,7 @@ const SelectionOverlay = (props: {
           iconType={SessionIconType.Exit}
           iconSize={SessionIconSize.Medium}
           onClick={onCloseOverlay}
-          theme={theme}
+          theme={useTheme()}
```

Comment:
no need to pass the theme at all now that most of the content is under the Theme Provider